### PR TITLE
Elevator

### PR DESCRIPTION
**Pull request by Lewis-Seiden**

### CODE REVIEW

File: deploy/choreo/amp 4 local.1.traj
Comment (Awesomeplayer165) [inline deploy/choreo/amp 4 local.1.traj:1]: why are choreo trajectory files added here? shouldn't they be in a different branch than `elevator`?

File: simgui-ds.json [hunk 1]
```diff
@@ -96,8 +96,7 @@
   ],
   "robotJoysticks": [
     {
-      "guid": "78696e70757401000000000000000000",
-      "useGamepad": true
+      "guid": "Keyboard0"
```
Comment (Awesomeplayer165) [inline simgui-ds.json:99]: im seeing this through many PRs. consider changing this in main?

File: src/main/java/frc/robot/subsystems/elevator/ElevatorIO.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.elevator;
+
+import org.littletonrobotics.junction.AutoLog;
+
+/** Add your docs here. */
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/subsystems/elevator/ElevatorIO.java:9]: not complete? need to add docs

File: src/main/java/frc/robot/subsystems/elevator/ElevatorIOReal.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.GravityTypeValue;
+
+/** Add your docs here. */
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/subsystems/elevator/ElevatorIOReal.java:15]: add some docs pls

**Review by Awesomeplayer165 — CHANGES_REQUESTED** [review]
small things

File: deploy/choreo/amp 4 local.1.traj
Comment (Lewis-Seiden) [reply deploy/choreo/amp 4 local.1.traj:1]: that is strange

File: simgui-ds.json (hunk 1 above)
Comment (Lewis-Seiden) [reply simgui-ds.json:99]: see my note in carriage pr

**Review by Awesomeplayer165 — APPROVED** [review]
LGTM

**Pull request merged**